ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -328,15 +330,25 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                                 )}
                                         </div>
                                         <SlidingPanel isShown={this.state.groupedResources.length > 0} onClose={() => this.closeGroupedNodesPanel()}>
-                                            <ApplicationResourceList
-                                                onNodeClick={fullName => this.selectNode(fullName)}
-                                                resources={this.state.groupedResources}
-                                                nodeMenu={node =>
-                                                    AppUtils.renderResourceMenu({...node, root: node}, application, tree, this.appContext, this.appChanged, () =>
-                                                        this.getApplicationActionMenu(application, false)
-                                                    )
-                                                }
-                                            />
+                                            <div style={{marginTop: '1.25em'}}>
```

Comment:
I know there are places where we mix styles in the code, but maybe for this one, we could add it to the scss file?